src/cmd/input.go

```diff
@@ -11,14 +11,16 @@ import (
 var dataFile string
 
 func readInputConfig() {
+	viper.SetConfigType("yaml")
 	switch dataFile {
 	case ".":
+		// TODO: does this block ever actually ever run?
```

Comment:
Somebody could be using this case.
Setting `-f` to `.` is called out in the command help docs for `opslevel create --help` ([create.go](https://github.com/OpsLevel/cli/blob/main/src/cmd/create.go#L17)) and `opslevel update --help` ([update.go](https://github.com/OpsLevel/cli/blob/main/src/cmd/update.go#L17)).

Reply:
I could not find a way to get it to trigger though, at least on bash/zsh. IIRC I ran into this problem before, passing `.` as an argument in CLI's is a bad idea 😅 

I don't think we should remove it immediately, but we should look into it because if this branch never gets called the `data.yaml` stuff is confusing.